week1-2/style.css

```
@@ -0,0 +1,137 @@
+.header {
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  background-color: skyblue;
+  border-radius: 20px;
+}
+
+.all_my_amount {
+  display: flex;
+  flex-direction: column;
+}
+
+.my_amount {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+
+.my_amount_title {
+  font-size: 5rem;
+  margin-bottom: 5px;
+  border-bottom: 8px solid #4390e1;
+}
+
+.my_amount_sub {
+  font-size: 30px;
+}
+
+.amount_detail_container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 30px;
+  gap: 50px;
+}
+
+.amount_detail_plus_container,
+.amount_detail_minus_container {
+  text-align: center;
+  display: flex;
+}
+
+.amount_detail_plus_container {
+  color: blue;
+}
+
+.amount_detail_minus_container {
+  color: red;
+}
+
+/* 내역 리스트 */
+
+.pay_detail {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 5px;
+}
+
+.date_btn {
+  margin: 20px;
+}
+
+.pay_detail_list {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 8px solid #4390e1;
+}
+
+.pay_detail_list > p {
+  font-weight: bold;
+}
+
+.pay_detail_contents {
+  width: 100%;
+  height: 500px;
+  overflow-y: scroll;
+}
+
+.pay_detail_item {
+  display: flex;
+  width: 100%;
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  font-size: 30px;
+}
+
+li {
+  display: flex;
+  justify-content: space-between;
+  margin: 5px;
+  width: 96%;
+  border-radius: 30px;
+  background-color: rgb(221, 153, 248);
+}
+
+.pay_detail_btn {
+  display: flex;
+  border: none;
+}
+
+/* 사용 내역 컨텐츠 */
+.pay_detail_content {
+  font-weight: bold;
+}
+
+.pay_detail_amount_spend {
+  color: red;
+}
+
+.pay_detail_amount_income {
+  color: blue;
+}
```

위에 있는 `amount_detail_plus_container`와 `amount_detail_minus_container` 그리고 이 친구들은 전부 color 스타일링 만을 위해서 존재하는 것 같은데, 이런 경우에는 두가지 방법이 있을 것 같아!
1️⃣ 한번에 쓰기!
```suggestion
.pay_detail_amount_spend, amount_detail_minus_container {
  color: red;
}

.pay_detail_amount_income, amount_detail_plus_container {
  color: blue;
}
```

2️⃣ 아래처럼 줘서 각 색마다 같은 클래스를 주기
```suggestion
.spend {
  color: red;
}

.income {
  color: blue;
}
```